web/src/components/shared/Header/Navigation.tsx

```diff
@@ -1,46 +1,76 @@
-import { FC } from 'react';
+import { FC, useEffect, useState } from 'react';
 import { Box } from 'web/src/components/ui/Box';
 import { Stack } from 'web/src/components/ui/Stack';
-import { Links, RenderNavLinkComponent } from 'web/src/components/shared/sharedTypes';
-import * as s from './Navigation.css';
+import { Links, LinkTabs, RenderNavLinkComponent } from 'web/src/components/shared/sharedTypes';
+import { NavigationItem } from './NavigationItem';
 
 interface Props {
   renderNavLinkComponent: RenderNavLinkComponent;
   navLinks: Links;
+  pathname?: string | undefined;
 }
 
-export const Navigation: FC<Props> = ({ navLinks, renderNavLinkComponent }) => {
+export const findActiveTabByPathname = (navLinks: Links, pathname: string | undefined) => {
+  if (!pathname) {
+    return undefined;
+  }
+
+  return navLinks.find(
+    (item) =>
+      item.type === 'tab' &&
+      item.tabs.findIndex((tab) => tab.to === pathname || pathname.indexOf(tab.to) > -1) > -1,
```

Comment:
```suggestion
      item.tabs.some((tab) => tab.to === pathname || pathname.startsWith(tab.to)),
```
Думаю с `startsWith` будет лучше совпадение. Нам же не нужно совпадение путей где-то по середине?